pkg/asset/machines/nutanix/machinesets.go

```diff
@@ -24,57 +24,105 @@ func MachineSets(clusterID string, config *types.InstallConfig, pool *types.Mach
 
 	platform := config.Platform.Nutanix
 	mpool := pool.Platform.Nutanix
-
 	total := int32(0)
 	if pool.Replicas != nil {
 		total = int32(*pool.Replicas)
 	}
+
 	var machinesets []*machineapi.MachineSet
-	provider, err := provider(clusterID, platform, mpool, osImage, userDataSecret)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to create provider")
+	numOfFDs := int32(len(mpool.FailureDomains))
+	len1 := numOfFDs
```

Comment:
Can we name this variable something meaningful?

Reply:
changed.